layouts/index.html

```diff
@@ -618,7 +618,7 @@ <h2 class="accordion-header" id="flush-heading-7">
                         </h2>
                         <div id="flush-collapse-7" class="accordion-collapse collapse " aria-labelledby="flush-heading-7" data-parent="#accordionFAQ">
                             <div class="accordion-body">
-                                {{ i18n "Yes, AlmaLinux OS is a 1:1 binary compatible fork of RHEL®, which means that your applications and services will remain compatible. For that reason, you can rapidly migrate any number of servers with just one command." }}
+                                {{ i18n "As stated, AlmaLinux is ABI compatible with RHEL®, which means that your applications and services should be completely interoperable. You can rapidly migrate any number of servers using the [migration tool](https://github.com/AlmaLinux/almalinux-deploy)." }}
```

Comment:
```suggestion
                                {{ i18n "Since AlmaLinux is ABI compatible with RHEL®, your applications and services should be completely interoperable. You can rapidly migrate any number of servers using the [migration tool](https://github.com/AlmaLinux/almalinux-deploy)." }}
```

"As stated" seems a bit dick-ish IMO and with translations could get even worse for other languages.